client/namespace.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package client
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/cockroach/keys"
+	"github.com/cockroachdb/cockroach/proto"
+)
+
+// TODO(pmattis):
+//
+// - This file contains the experimental Cockroach table-based interface. The
+//   API will eventually be moved into db.go, but is collected here during
+//   initial development. Much of the implementation will eventually wind up on
+//   the server using new table-based requests to perform operations.
+//
+// - Add support for namespaces. Currently namespace ID 0 is hardcoded.
```

Comment:
This has now all been moved to `table.go` and the tests moved to `table_test.go`.